packages/dynamite/lib/src/openapi_builder.dart

```diff
@@ -1549,7 +1549,10 @@ TypeResult resolveType(
                               ],
                             ],
                             if (schema.oneOf != null) ...[
-                              "assert([${fields.values.join(',')}].where((final x) => x != null).length == 1, 'Need oneOf for \$data');",
+                              "assert([${fields.values.join(',')}].where((final x) => x != null).length >= 1, 'Need oneOf for \$data');",
+                            ],
+                            if (schema.anyOf != null) ...[
+                              "assert([${fields.values.join(',')}].where((final x) => x != null).length >= 1,  'Need anyOf for \$data');",
```

Comment:
I thought I remembered that `anyOf` means any number of matches, even if it is zero, but I can't find it in the spec :woman_shrugging:

Reply:
Nope any of has at least one match.

We already talked about that
https://github.com/provokateurin/nextcloud-neon/issues/193#issuecomment-1430138395

Reply:
Alright, I was looking at https://spec.openapis.org/oas/latest.html which is the official spec (swagger is theoretically exactly the same, but they sometimes have better explanations or examples).